lib/widgets/inbox.dart

```diff
@@ -245,6 +245,7 @@ abstract class _HeaderItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Material(
+      // TODO(#95) need dark-theme color
```

Comment:
Oh weird; I think maybe it got this way because I added the "Related" line hastily and I forgot there was a commit-message body. There was a commit-message body, but I guess I accidentally lumped it in with the other text in the editor that is not commit-message body:

```
ui [nfc]: Comment with TODO(#95) where we need dark-theme colors

Related: #95
In these places, I was unable to find a specific dark-theme color by
consulting web or the Figma (as it is now).

# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# Date:      Sun May 19 15:44:50 2024 -0700
#
# interactive rebase in progress; [etc.]
```

Reply:
I usually put "Foo: bar"-style lines at the ends of commit messages; do you think we should start putting them at the top of the body?

Reply:
No, at the end is good.